src/main/java/it/gov/pagopa/debtposition/repository/specification/PaymentPositionByServiceType.java

```diff
@@ -0,0 +1,20 @@
+package it.gov.pagopa.debtposition.repository.specification;
+
+import it.gov.pagopa.debtposition.entity.PaymentPosition;
+import it.gov.pagopa.debtposition.model.enumeration.ServiceType;
+import lombok.NoArgsConstructor;
+import org.springframework.data.jpa.domain.Specification;
+
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Predicate;
+import javax.persistence.criteria.Root;
+
+@NoArgsConstructor
+public class PaymentPositionByServiceType implements Specification<PaymentPosition> {
+
+    @Override
+    public Predicate toPredicate(Root<PaymentPosition> root, CriteriaQuery<?> query, CriteriaBuilder cb) {
+        return cb.notEqual(root.get("serviceType"), ServiceType.WISP);
```

Comment:
Can you provide to pass and use a `ServiceType` parameter, like [this specification](https://github.com/pagopa/pagopa-debt-position/blob/820f15c218fe38bdcec8659107f1ea751ba50357/src/main/java/it/gov/pagopa/debtposition/repository/specification/PaymentPositionByStatus.java#L15)? In this way, we can generalize this specification and also use it for other cases than WISP.